src/types.ts

```diff
@@ -223,4 +223,5 @@ export type TPermission =
   | "dashboard.access"
   | "laboratories.access"
   | "patients.access"
-  | "opds.access";
+  | "opds.access"
+  | "admin.access";
```

Comment:
please keep in sync with `/mockServer/fixtures/permissionList.js` as described in the comment or deprecate the comment